src/interfaces/widgets/module.py

```diff
@@ -0,0 +1,300 @@
+"""
+
+    This module generate serveral combination frame based object.
+
+"""
+
+from PyQt5.QtWidgets import QWidget
+from PyQt5.QtCore import Qt
+
+from .button import Button, RadioButton
+from .frame import Frame
+from .groupbox import GroupBox
+from .label import Label
+from .layout import HorizontalLayout
+from .lineedit import LineEdit
+from .spacer import HorizontalSpacer
+
+
+class BaseTwoLabelFrame(Frame):
```

Comment:
What is a BaseTwoLabelFrame?

Reply:
This class also looks like it contains most of the same functionality of `BaseLabelFrame`. Is it possible to have this inherit from `BaseLabelFrame`?

Reply:
There's no module named `BaseLabelFrame`. I named it `BaseTwoLabelFrame` is just because this frame contains two labels. We got some widget have similar structure (i.e. config, sectionTitle, etc.) If you think we should just go with the name `BaseLabelFrame`, I can change it.